frontend/src/views/Datasets/DatasetConsoleAccess.js

```diff
@@ -47,16 +47,14 @@ const DatasetConsoleAccess = (props) => {
           {dataset.IAMDatasetAdminRoleArn}
         </Typography>
       </CardContent>
-      {!dataset.imported && (
-        <CardContent>
+      <CardContent>
           <Typography color="textSecondary" variant="subtitle2">
             KMS alias
           </Typography>
           <Typography color="textPrimary" variant="body2">
             {`arn:aws:kms:${dataset.region}:${dataset.AwsAccountId}/alias:${dataset.KmsAlias}`}
           </Typography>
```

Comment:
I just checked that the dataset.KmsAlias will never be a None value, by default we write `KmsAlias='undefined'`